agent/agent_worker_test.go

```diff
@@ -255,7 +255,7 @@ func TestAgentWorker_Start_AcquireJob_Pause_Unpause(t *testing.T) {
 	// Pre-register the agent.
 	const agentSessionToken = "alpacas"
 	agent := server.AddAgent(agentSessionToken)
-	agent.PingHandler = func() (api.Ping, error) {
+	agent.PingHandler = func(req *http.Request) (api.Ping, error) {
```

Comment:
For implementations that don't look at the arg, the name can be omitted

```suggestion
	agent.PingHandler = func(*http.Request) (api.Ping, error) {
```